humanize-duration.js

```diff
@@ -490,7 +490,14 @@
 
       // What's the number of full units we can fit?
       if (i + 1 === len) {
-        unitCount = ms / unitMS
+        if (options.maxDecimalPoints) {
```

Comment:
This should be `options.hasOwnProperty('maxDecimalPoints')` to prevent when `maxDecimalPoints === 0`.